src/index.ts

```diff
@@ -29,6 +30,8 @@ router.put("/roles/add", addGroupRoleHandler);
 
 router.get("/discord-members", getMembersInServerHandler);
 
+router.get("/member/:id", getGuildMemberDetailsHandler);
```

Comment:
can we try adding it to the route `/discord-members` and provide a URL parameter with id something like `/discord-members?id=123` and then call your service?

Reply:
ID's or unique identifier are to be sent as path params always. That's a standard convention